src/containers/MyNdla/Folders/FolderDeleteModal.tsx

```diff
@@ -0,0 +1,49 @@
+/**
+ * Copyright (c) 2022-present, NDLA.
```

Comment:
Denne fila ser veldig ny ut.
```suggestion
 * Copyright (c) 2023-present, NDLA.
```

Reply:
Vurder om denne skal oppdateres.